special-pages/pages/duckplayer/app/features/error-detection.js

```diff
@@ -18,11 +21,19 @@ export class ErrorDetection {
     /** @type {CustomErrorSettings} */
     options;
 
+    /** @type {Logger} */
+    logger;
+
     /**
      * @param {CustomErrorSettings} options
      */
     constructor(options) {
         this.options = options;
+        this.errorSelector = options?.settings?.youtubeErrorSelector || '.ytp-error';
```

Comment:
Keeping `.ytp-error` as default here so that I can [carry on with the move to privacy-config](https://app.asana.com/1/137249556945/project/1203249713006009/task/1210483318529990?focus=true) with minimal risk